lib/iris/analysis/stats.py

```diff
@@ -111,7 +120,8 @@ def pearsonr(cube_a, cube_b, corr_coords=None):
     For example providing two time/altitude/latitude/longitude
     cubes and corr_coords of 'latitude' and 'longitude' will result
     in a time/altitude cube describing the latitude/longitude
-    (i.e. pattern) correlation at each time/altitude point.
+    (i.e. pattern) correlation at each time/altitude point. Area
+    weights may be set using :func:`iris.analysis.cartography.area_weights`
```

Comment:
Whilst this is true, it makes it sounds like that is the only way weights might be generated. It would be more appropriate to just mention that weights may be provided if required. You can mention area weighting as an example where the `weights` keyword itself is documented if you like.